frontend/src/components/Collections/components/CollectionsGrid/components/CollectionRow/components/Popover/style.ts

```diff
@@ -0,0 +1,12 @@
+import { GRAY, PT_GRID_SIZE_PX } from "src/components/common/theme";
+import styled from "styled-components";
+import { textClippingCSS } from "../../../common/style";
+
+export const FieldValues = styled.div`
+  ${textClippingCSS}
+  color: ${GRAY.A};
+`;
+
+export const ContentWrapper = styled.div`
+  padding: ${2 * PT_GRID_SIZE_PX}px;
```

Comment:
Harley wants us to use `PT_GRID_SIZE_PX` as base for styling as much as possible. Such as margin, padding, etc.